src/server/index.jsx

```diff
@@ -142,12 +145,12 @@ const injectDefaultCacheHeader = (req, res, next) => {
   const defaultMaxAge = getDefaultMaxAge(req);
   const maxAge =
     req.originalUrl.indexOf('/topics/') !== -1
-      ? defaultMaxAge * 2
+      ? defaultMaxAge * 4
```

Comment:
Be good to monitor this when it goes out, we should see a material drop in topic requests I would hope! Fwiw I'd imagine up to 10 minutes might be okay for topics and given they are low traffic that might make the biggest difference?

Reply:
I'm nudging it up each time to see what the sweet spot is